code/style.css

```diff
@@ -1,5 +1,99 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+.image1 {
+height: 200px;
+padding: 0cm;
+width: 100%;
+}
```

Comment:
Some indentation missing here:
```
.image1 {
  height: 200px;
  padding: 0cm;
  width: 100%;
}
```

Reply:
Also, consider using `px` instead of `cm`. Read more [here](https://www.w3schools.com/cssref/css_units.asp).